src/BrazeKit-dev.js

```diff
@@ -965,6 +991,18 @@ var constructor = function () {
         }
     }
 
+    function decodeSubscriptionGroupMappings(subscriptionGroupSetting) {
+        var subscriptionGroupIds = {}; 
+      	var decodedSetting = subscriptionGroupSetting.replace(/&quot;/g, '"');
+        var parsedSetting = JSON.parse(decodedSetting);
+        for (let subscriptionGroupMap of parsedSetting) {
+            var key = subscriptionGroupMap.map;
+            var value = subscriptionGroupMap.value;
+            subscriptionGroupIds[key] = value;
+        }
+        return subscriptionGroupIds;
```

Comment:
@mmustafa-tse I realize, we should actually put a try/catch block here just to be extra safe